spec/controllers/alaveteli_pro/stripe_webhooks_controller_spec.rb

```diff
@@ -312,7 +312,10 @@
 
     describe 'a customer moves to a new billing period' do
       let(:stripe_event) do
-        StripeMock.mock_webhook_event('customer.subscription.updated-renewed')
+        _event =
+          StripeMock.mock_webhook_event('customer.subscription.updated-renewed')
+        _event['data']['previous_attributes']['latest_invoice'] = 'inv_1234'
```

Comment:
Can just update the fixture instead of munging a value in?  https://github.com/mysociety/alaveteli/blob/develop/spec/fixtures/stripe_webhooks/customer.subscription.updated-renewed.json#L66-L69

Reply:
good point - fixed in 5f9be7c